fighters/link/src/acmd/tilts.rs

```diff
@@ -7,39 +7,28 @@ unsafe fn attack_s3_s(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
     frame(lua_state, 1.0);
-    if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 0.476);
-    }
+    FT_MOTION_RATE(fighter, 1.0/(3.0-1.0));
     frame(lua_state, 3.0);
-    if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 0.80);
-    }
+    FT_MOTION_RATE(fighter, 8.0/(13.0-3.0));
     frame(lua_state, 13.0);
-    if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 1.000);
-    }
+    FT_MOTION_RATE(fighter, 1.0);
     frame(lua_state, 15.0);
-    if is_excute(fighter) {
-        FT_MOTION_RATE(fighter, 2.0);
-    }
+    FT_MOTION_RATE(fighter, 1.0/(15.5-15.0));
```

Comment:
math works out the same but this should technically be `FT_MOTION_RATE(fighter, 2.0/(16.0-15.0));`